src/runtime/cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -222,12 +221,6 @@ vhost_user_store_path = "@DEFVHOSTUSERSTOREPATH@"
 # used for 9p packet payload.
 #msize_9p = @DEFMSIZE9P@
 
-# If true and vsocks are supported, use vsocks to communicate directly
-# with the agent and no proxy is started, otherwise use unix
-# sockets and start a proxy to communicate with the agent.
-# Default false
-#use_vsock = true
```

Comment:
Shouldn't we default to "true" and warn if the option is set to false, instead of rejecting the option entirely?

Alternatively, start a document that lists all options that are removed from 2.0 so that people can proactively edit their configuration files.

Reply:
We are dropping serial port support in 2.0, -- that's why `use_vsock` is just removed instead of switching to `true` as default.